blocks/covid-19-accordion/covid-19-accordion.js

```diff
@@ -0,0 +1,139 @@
+ * @param pictureElement
+ * @param extractedInnerHTML
+ * @returns {HTMLDivElement}
+ */
+function createBanner(pictureElement, extractedInnerHTML) {
+  const banner = document.createElement('div');
+  banner.classList.add('banner');
+
+  const info = document.createElement('div');
+  info.classList.add('info');
+  info.innerHTML = extractedInnerHTML;
+  banner.appendChild(info);
+
+  const icon = document.createElement('div');
+  icon.classList.add('icon');
+  icon.innerHTML = pictureElement.outerHTML;
+  banner.appendChild(icon);
+
+  return banner;
+}
+
+/**
+ * Creates dividers with close buttons for the expanded content of accordion
+ * @returns {HTMLDivElement}
+ */
+function createDividerWithCloseButton() {
+  const newDiv = document.createElement('div');
+  newDiv.classList.add('covid-19-accordion', 'custom-divider');
+
+  const hr = document.createElement('hr');
+  newDiv.appendChild(hr);
+
+  const closeButton = document.createElement('button');
+  closeButton.classList.add('covid-19-accordion', 'close-button');
+  // empty as we fill in from after pseudo element
+  closeButton.innerHTML = '';
+  closeButton.addEventListener('click', () => {
+    const accordion = document.querySelector('.covid-19-accordion .container .details');
+    const button = document.querySelector('#expand-accordion');
+
+    accordion.classList.remove('expanded');
+    button.setAttribute('aria-expanded', 'false');
+  });
+  newDiv.appendChild(closeButton);
+
+  return newDiv;
+}
+
+/**
+ * Sets up the first button to expand the accordion and adds a close button to the accordion
+ * @param button
+ */
+function setupFirstButton(button) {
+  button.setAttribute('id', 'expand-accordion');
+  button.setAttribute('aria-expanded', 'false');
+  button.setAttribute('aria-controls', 'details');
+  button.addEventListener('click', () => {
+    const accordion = document.querySelector('.covid-19-accordion .container .details');
+    const existingDivider = accordion.querySelector('.custom-divider');
+
+    if (!existingDivider) {
+      const newDiv = createDividerWithCloseButton();
+      accordion.prepend(newDiv);
+    }
+
+    accordion.classList.toggle('expanded');
+    if (accordion.classList.contains('expanded')) {
+      button.setAttribute('aria-expanded', 'true');
+    } else {
+      button.setAttribute('aria-expanded', 'false');
+    }
+  });
+}
+
+/**
+ * Creates a div with the extracted links from the first row of document
+ * @param linksArray
+ * @returns {HTMLDivElement}
+ */
+function createLinks(linksArray) {
+  const links = document.createElement('div');
+  links.classList.add('links');
+
+  linksArray.forEach((link, index) => {
+    const button = document.createElement('button');
+    button.textContent = link.textContent;
+
+    if (index === 0) {
+      setupFirstButton(button);
+    } else {
+      // eslint-disable-next-line no-return-assign
+      button.onclick = () => (window.location.href = link.href);
```

Comment:
do you need this? since it is a link clicking it will take you anyway to the new page. 
in theory you just need to transform the first link into button, all the others can remain links styled as buttons.

Reply:
it is needed as those are "new" buttons created on the fly.

Reply:
@AlexRRR :
I meant that you can keep the anchor (`<a>`) elements as they are and just style them the same way as the buttons.
Like so:
* https://github.com/fieitas/website/pull/13/files#diff-a846cfae8bbf555aaf18cdfc1cb374b416e1bd3566108643c47bafe7844faf37R88
* https://github.com/fieitas/website/pull/13/files#diff-387354b9540fc5bdca52b4378cd1dcc32837f4cf0e231946dc21f68c34ea3c6eR27-R28

It means less JS processing and also my understanding is that semantically:
* Buttons are for anything that triggers something on the same page (like here, expansion of an accordion)
* Anchors are for taking you to a new page
Of course this is not set in stone :) Like you can have anchors taking you to different sections of the same page, etc.